lib/screens/request_board_screen.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:cobe_hive_mobile_app/app_colors.dart';
+import 'package:cobe_hive_mobile_app/widgets/expandable_fab.dart';
+import 'package:cobe_hive_mobile_app/widgets/request_approval_list.dart';
+import 'package:cobe_hive_mobile_app/widgets/request_board_header.dart';
+import 'package:flutter/material.dart';
+
+class RequestBoardScreen extends StatelessWidget {
+  const RequestBoardScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      resizeToAvoidBottomInset: false,
+      body: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 18),
+        child: Column(
+          children: [
+            const Padding(
+              padding: EdgeInsets.only(bottom: 40, top: 18),
+              child: RequestBoardHeader(),
+            ),
+            Row(
+              children: [
+                const Text(
+                  'Pending',
+                  style: TextStyle(
+                    fontSize: 24,
+                    fontWeight: FontWeight.w500,
+                  ),
+                ),
+                const Expanded(
+                    child: SizedBox(
+                  height: 30,
+                )),
+                TextButton(
+                  onPressed: () {
+                    Navigator.pushNamed(context, '/request-board-approved');
+                  },
+                  child: const Text(
+                    'See Approved',
+                    style: TextStyle(
+                      color: AppColors.primary,
+                      fontSize: 17,
+                    ),
+                  ),
+                )
+              ],
+            ),
+            const RequestApprovalList(),
+          ],
+        ),
+      ),
+      floatingActionButton: const ExpandableFAB(),
```

Comment:
`child` and other params that build or return `Widget`'s should be the last paramters in the list.

`floatingActionButton ` also returns a `Widget`. However, it is one line of code long. It is diffucult for the reader to spot it. It would be much nicer id it is above `child` paramter.